### DISCUSSION TITLE
How do I configure which-key with rocks.nvim?

### DISCUSSION THREAD

**Discussion by Folaht** [lumen-oss/rocks.nvim#594] [Q&A]
I'll ask on the which-key discussion board.
It's more suitable there.

**Comment by mrcjkb** [comment]
Hey :wave: 

The latest which-key doesn't require any configuration for the default setup.
For configuring plugins in general, see [this response](https://github.com/nvim-neorocks/rocks.nvim/discussions/139#discussioncomment-8429847).
